File: benchmark/comparison/results/blind_ts/12.ts
function power(base: number, exp: number): number {
  let result = 1;
  for (let i = 0; i < exp; i++) result *= base;
  return result;
}

console.log(power(2, 0));
console.log(power(2, 10));
console.log(power(3, 4));
console.log(power(5, 3));
console.log(power(1, 100));

File: benchmark/comparison/results/blind_py/12.py
def power(base, exp):
    result = 1
    for _ in range(exp):
        result *= base
    return result

print(power(2, 0))
print(power(2, 10))
print(power(3, 4))
print(power(5, 3))
print(power(1, 100))
